versioned_docs/version-23.10/releases/lifecycle.md

```diff
@@ -49,14 +49,14 @@ Centreon 20.04 and the second phase of Centreon 20.10.
 
 ### From version 21.10
 
-From version 21.10, versions are supported for 2 years. The lifecycle of a version is divided into 2 phases of 12 months each.
+From version 21.10, versions have been supported for two years. The lifecycle of a version is divided into two phases of 12 months each.
```

Comment:
```suggestion
Since version 21.10, versions are supported for two years. The lifecycle of a version is divided into two phases of 12 months each.
```